packages/discovery-provider/ddl/migrations/0080_track_downloads.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE IF NOT EXISTS track_downloads (
+    txhash VARCHAR NOT NULL,
+    blocknumber INTEGER NOT NULL,
+    parent_track_id INTEGER NOT NULL,
+    track_id INTEGER NOT NULL,
+    user_id INTEGER,
+    created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    CONSTRAINT track_downloads_blocknumber_fkey FOREIGN KEY (blocknumber) REFERENCES blocks("number") ON DELETE CASCADE,
```

Comment:
do we have these constraints elsewhere now? especially the on delete cascade part. just curious if this is in line with the current strategy for these tables

Reply:
Yeah we do it for eg track routes i believe and others - cascade deletes helps w/ reverts and replays. CC: @isaacsolo to confirm I set this up correctly